antlr-kotlin-target/src/main/resources/org/antlr/v4/tool/templates/codegen/Kotlin/Kotlin.stg

```diff
@@ -738,16 +738,15 @@ CaptureNextTokenType(d) ::= "<d.varName> = _input!!.LA(1)"
 
 StructDecl(struct,ctorAttrs,attrs,getters,dispatchMethods,interfaces,extensionMembers)
 	::= <<
-open class <struct.name> : <if(contextSuperClass)><contextSuperClass><else>ParserRuleContext<endif><if(interfaces)> : <interfaces; separator=", "><endif> {
+open class <struct.name>(parent: ParserRuleContext?, invokingState: Int) : <if(contextSuperClass)><contextSuperClass><else>ParserRuleContext<endif>(parent, invokingState)<if(interfaces)> : <interfaces; separator=", "><endif> {
```

Comment:
Maybe it is here the problem that is causing the compilation issue. It involves some constructor not being called

Reply:
Why do we need to always pass these two params to the constructor?

Reply:
I think I was a too quick to change this - I'm going to back out this change and the one below that goes with it.